### PR TITLE
Added test for flatattrs filter

### PR DESCRIPTION
**Pull request by rapilabs**
I've simply assumed that the current behaviour is correct and have tested against flattattrs output.

### CODE REVIEW

**Comment by rapilabs** [comment]
So tests are failing for requirement Django<1.5 due to double-quotes being entity encoded.  I've never used these older versions of Django, not sure whether this is correct behaviour or not.

File: tests/test_filters.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    def test_flatattrs(self):
+        tmpl = get_template('flatattrs')
+        self.ctx['a_dict'] = {
+            'a': 'aye',
+            'b': 'bee',
+            'c': 'cee',
+        }
+        output = tmpl.render(self.ctx)
+
+        if VERSION[1] >= 5:
```
Comment (funkybob) [inline tests/test_filters.py]: Make this explicitly check VERSION[:2] >= (1, 5)

**Comment by schinckel** [comment]
I was wondering if this was just that the result of the old version of django's flatattrs was not marking the string as safe?

If so, we could fix that.

**Comment by schinckel** [comment]
Yep, it seems that newer versions of django call mark_safe on the result.

https://github.com/django/django/blob/master/django/forms/utils.py#L26

Even though that docstring is misleading, it does eventually call mark_safe, so I believe it is safe for us to do so in 1.4.

Since I'm possibly the only user of sniplates under 1.4, I'm deciding not to implement the other features of flatatt.

**Pull request merged**